catalogue/mpi/pdq/user-guide/quickstart.md

```diff
@@ -0,0 +1,294 @@
+## Quickstart
+
+To follow along with this quickstart you'll need:
+
+- a tool for sending sending HTTP requests, e.g. Postman, SOAP UI
+- access to a test endpoint, e.g. the public sandbox
+
+There are a number of different combinations of fields that can be combined to search the MPI. A sample of these will be detailed with examples below. 
+
+## HTTP Header values
+
+To POST any search requests successfully you will need to set the following API Header values. There would be entered under the [Headers tab](https://learning.postman.com/docs/sending-requests/requests/#configuring-request-headers) in Postman or as a [custom HTTP header](https://www.soapui.org/docs/functional-testing/teststep-reference/http-request/headers/) in SOAPUI.
+
+| Field                     |  Value
+|---------------------------|:-------------------------------------------------------------|
+| Ocp-Apim-Subscription-Key |  <your subscription key>*                                    |
```

Comment:
```suggestion
| Ocp-Apim-Subscription-Key |  \<your subscription key\>*                                    |
```
Need to escape things that look like html in your markdown

Reply:
thumbs up for that one.